src/hooks/use-heatmap/use-heatmap.js

```diff
@@ -0,0 +1,10 @@
+import { useState } from 'react';
```

Comment:
This hook seems a bit overkill. You could simply call the `getHeatmapFromPosts` in the component.

But tbh, I'd rather store the mapped data instead of the raw posts from the response in the `useFetchPosts` hook.

Reply:
Definitely overkill. Yeah, I agree that it's unnecessary to store the raw posts and the mapped data can be stored instead. In that case, I should preserve all the post data during the mapping, that way it is accessible to the `PostsTable` component in the next task.